hedy.py

```diff
@@ -1235,6 +1230,24 @@ def turn(self, args):
             raise exceptions.InvalidArgumentTypeException(command=Command.turn, invalid_type='', invalid_argument=arg,
                                                           allowed_types=get_allowed_types(Command.turn, self.level))
 
+    def make_turn(self, parameter):
+        return self.make_turtle_command(parameter, Command.turn, 'right', False)
+
+    def make_forward(self, parameter):
+        return self.make_turtle_command(parameter, Command.forward, 'forward', True)
+
+    def make_turtle_command(self, parameter, command, command_text, add_sleep):
+        variable = self.get_fresh_var('trtl')
+        transpiled = textwrap.dedent(f"""\
+            {variable} = {parameter}
+            try:
+              {variable} = int({variable})
+            except ValueError:
```

Comment:
Ow smart way to pass runtime errors to the front-end! This can be used on other places too I think (will have to test it more extensively)